portfolio/src/main/java/com/google/sps/comment/data/Comment.java

```diff
@@ -34,6 +38,7 @@ private Comment(long id, String commenter, String content, long timestamp) {
     public Entity toEntity(String key) {
```

Comment:
This param was not used and the local constant is used instead?

Reply:
what a stupid bug... fixed it